{crossTime: 0.5, errorAng: 1, errorCorrectionDis: 150, errorMoibleAng: 2, errorMoibledis: 1000,
  errorRotate0: 1, errorRotate90: 89, maxVel: 10.0, minVel: 18, rotateSlowAng: 20,
  rotateYaw: 45, velYaw: 16}
